recipes/micronota/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+pip install burrito click
```

Comment:
burrito is not conda installable (at least not in default channel) and
click of conda is in low version. Is there a way to set pip installable dep
in requirements section?

On Wed, Jan 13, 2016 at 8:25 PM, Jorge Cañardo Alastuey <
notifications@github.com> wrote:

> In recipes/micronota/build.sh
> <https://github.com/biocore/conda-recipes/pull/56#discussion_r49686569>:
>
> > @@ -0,0 +1,4 @@
> > +#!/bin/bash
> > +
> > +pip install burrito click
>
> Is burrito and click in the packaged micronota? I thought it had to be in
> the requirements section or otherwise it those dependencies wouldn't be
> available when installing.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/conda-recipes/pull/56/files#r49686569>.
>